mmv1/provider/core.rb

```diff
@@ -190,12 +190,72 @@ def compile_file_list(output_folder, files, file_template, pwd = Dir.pwd)
           Google::LOGGER.debug "Compiling #{source} => #{target}"
           file_template.generate(pwd, source, target, self)
 
+          add_hashicorp_copyright_header(output_folder, target)
```

Comment:
Do we also need to add this function to copy_file_list to copy files to downstream?

https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/provider/core.rb#L130

Reply:
Yes I believe so - thank you for spotting this!